xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/delete.vm

```diff
@@ -238,20 +238,20 @@
 #######################################################
 #macro(displayConfirmMessage)
   #if($xwiki.hasRecycleBin())
-    #if($services.refactoring.isAllowedToSkipTheRecycleBin())
-      <div class="xform">
+    #if($services.refactoring.isRecycleBinSkippingAllowed())
+      <div class="xform" id="skipRecycleBinForm">
         <dl>
           <dt>
-            $escapetool.xml($services.localization.render('core.recyclebin.wantToSkip'))
+            $escapetool.xml($services.localization.render('core.recycleBin.shouldSkipRecycleBin'))
```

Comment:
This doesn't seem to follow the convention. I took a lot of time to make proposals to you in my last replies. Could you reply to them and explain why you didn't like them and why you thought they were not following the conventions? Thx